### PR TITLE
Trev8939/elevation at point

### PR DESCRIPTION
**Pull request by TADraeseke**
Hi @priy1289 - Could you review this short sample when you have a moment?

Thanks!

### CODE REVIEW

File: java/elevation-at-point/README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+Knowing the elevation at a given point in a landscape can aid in navigation, planning and survey in the field.
+
+## How to use the sample
+
+Tap anywhere on the surface to get the elevation at that point.
+
+## How it works
+
+1. Create a `SceneView` and `Scene` with an imagery base map.
+1. Set an `ArcGISTiledElevationService` as the elevation source of the scene's base surface.
```
Comment (priy1289) [inline java/elevation-at-point/README.md]: Minor - should this be ArcGISTiledElevationSource?

File: java/elevation-at-point/src/main/java/com/esri/arcgisruntime/sample/elevationatpoint/MainActivity.java [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+      @Override public boolean onSingleTapConfirmed(MotionEvent motionEvent) {
+        // clear any existing graphics from the graphics overlay
+        graphicsOverlay.getGraphics().clear();
+        // get the tapped screen point
+        android.graphics.Point screenPoint = new android.graphics.Point(Math.round(motionEvent.getX()),
+            Math.round(motionEvent.getY()));
+        // convert the screen point to a point on the surface
+        Point surfacePoint = mSceneView.screenToBaseSurface(screenPoint);
+        // create a new graphic at the surface point and add it to the graphics overlay
+        Graphic surfacePointGraphic = new Graphic(surfacePoint, sphereSymbol);
```
Comment (priy1289) [inline java/elevation-at-point/src/main/java/com/esri/arcgisruntime/sample/elevationatpoint/MainActivity.java]: if surfacePoint is null, the app will crash. 
to test : Tap anywhere outside the elevation_image_service.

File: java/elevation-at-point/README.md (hunk 1 above)
Comment (TADraeseke) [reply java/elevation-at-point/README.md]: Good spot! Thanks!

File: java/elevation-at-point/src/main/java/com/esri/arcgisruntime/sample/elevationatpoint/MainActivity.java (hunk 2 above)
Comment (TADraeseke) [reply java/elevation-at-point/src/main/java/com/esri/arcgisruntime/sample/elevationatpoint/MainActivity.java]: Another good catch, thanks!

**Comment by TADraeseke** [comment]
Thanks @priy1289 - I've changed the point sphere into a 2D circle which now scales with zoom, so you shouldn't see the giant spheres anymore!

**Pull request merged**